### PR TITLE
Do not crash sed_unlock on SCALE

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/disk.py [hunk 1]
```diff
@@ -319,6 +319,9 @@ async def sed_unlock(self, disk_name, disk=None, _advconfig=None):
 
         rv = {'name': disk_name, 'locked': None}
 
+        if osc.IS_LINUX:
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/disk.py:322]: Shouldnt that be after sedutil-cli?

Comment (themylogin) [reply src/middlewared/middlewared/plugins/disk.py:322]: @william-gr do you want me to make sedutil debian package now then?

Comment (william-gr) [reply src/middlewared/middlewared/plugins/disk.py:322]: No, I thought we already had sedutil and only camcontrol was failing

Comment (william-gr) [reply src/middlewared/middlewared/plugins/disk.py:322]: Not sure whats the point in "fixing" this, then? if nothing is wokring anyway?

Comment (william-gr) [reply src/middlewared/middlewared/plugins/disk.py:322]: I think Waqar already has a ticket to make SED work

Comment (themylogin) [reply src/middlewared/middlewared/plugins/disk.py:322]: There is a report in community slack, pool.format_disks always calls sed_unlock_all and the crash prevents disks from formatting

Comment (themylogin) [reply src/middlewared/middlewared/plugins/disk.py:322]: Sorry, ticket about what?

Comment (william-gr) [reply src/middlewared/middlewared/plugins/disk.py:322]: Thought we had a ticket about wiping disk not working. If thats not the case, nvm me

Comment (themylogin) [reply src/middlewared/middlewared/plugins/disk.py:322]: You have not approved the ticket, is there anything else I am missing?

**Review by william-gr — APPROVED** [review]
No, simply forgot :)

**Pull request merged**